calls/forms.py

```diff
@@ -1,11 +1,10 @@
 from django import forms
 from django.conf import settings
-from django.forms import widgets
 from contacts.models import ContactListStatus
 from .models import (
     call_campaign_statuses_skip_list_validation,
     CallCampaign,
-    CallQuestion,
+    CallQuestion
```

Comment:
we dont really have a standard here, but i have been using the trailing comma on lists like this, which is acceptable in python. and you apparently do not. so we should agree at some point rather than keep changing each other's commas.

Reply:
I had stopped using them because I thought you weren't for a while, let's use them from now on.

Reply:
At first I didn't like it, then I decided it seems useful.